pkgs/applications/editors/jetbrains/plugins/update_plugins.py

```diff
@@ -380,6 +380,12 @@ def main():
 
     write_result(result)
 
+    # Commit the result
+    commitMessage = "jetbrains.plugins: update"
+    print("#### Committing changes... ####")
+    run(['git', 'add', PLUGINS_FILE], check=True)
+    run(['git', 'commit', '--file=-'], input=commitMessage.encode(), check=True)
```

Comment:
```suggestion
    run(['git', 'commit', f'{PLUGINS_FILE}' f'-m {ommitMessage.encode()}'], check=True)
```
you could probably do this in one line. Also why are you adding the commitMessage with the `input` parameter like above instead of doing `-m`?